server/src/lib/model/achievements.ts

```diff
@@ -85,44 +88,40 @@ const bonus_condition_sql = {
     `,
 };
 const db = getMySQLInstance();
-export default {
-  async earnBonus(type: AchievementType, args: any[]) {
-    let earned = false;
 
-    const [[res]] = await db.query(bonus_condition_sql[type], args);
-    const { win_bonus, bonus_winner } = res || {
-      win_bonus: false,
-      bonus_winner: '',
-    };
-    if (win_bonus && bonus_winner) {
-      const ret = await db.query(
-        `
-              INSERT INTO earn (achievement_id, client_id) VALUES ((SELECT id FROM achievements WHERE name = ?), ?)
-            `,
-        [type, bonus_winner]
-      );
-      earned = ret && ret[0] && ret[0].affectedRows > 0 ? true : false;
-    }
-    return earned;
-  },
```

Comment:
Oops! I forgot to update the import declarations. Fixed in https://github.com/mozilla/voice-web/pull/2272/commits/79cc0ab09efc5023b3ccb66713b3462cb9186366